modules/ROOT/pages/apicat-create-descriptor-file-manually.adoc

```diff
@@ -0,0 +1,226 @@
+= Creating or Updating Descriptor Files Manually
+
+The 'create-descriptor` and `update-descriptor` commands identify the APIs to catalog and output the essential information needed to catalog them. If you want to add more descriptor information, the most efficient approach is to first create or update the descriptor file using the CLI, and then manually update the file with information that cannot be automatically. Use the following information about the descriptor YAML schema to decide what to add. 
+
+If you already have `exchange.json` files associated with your API projects, you can reference the information in those files either in-line or by using an external reference. 
+
+[[descriptor-yaml]]
+== Descriptor YAML Schema
+
+Following are the YAML tags used to describe the APIs you want to catalog. 
+
+== Catalog Descriptor Line
+
+The descriptor file must start with the following:
+
+---
+
+ #%Catalog Descriptor 0.7
+
+---
+
+[[trigger-conditions]]
+== triggerConditions
+
+The `triggerConditions` section of the YAML file enables you to optionally set conditions to determine when to publish the set of APIs described in the descriptor file. If you add this section to the descriptor file, the APIs are published only when the `api-catalog publish-asset` command is run with `--trigger-criteria` options with matching values for each of the trigger conditions. This enables you use the same `api-catalog publish-asset` command in your CI/CD scripts for each of your branches or environments and pass parameters to the command to control whether the APIs are published. 
+
+You can set trigger conditions for specific APIs or for all APIs described in the descriptor file (global). If both types of conditions are used, the specific override the global. 
```

Comment:
"If both types of conditions are used, the specific override the global."

confirm this makes sense